projects/demo-playwright/tests/core/breakpoint-service/breakpoint-service.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Breakpoint service`, () => {
+    for (const {width, height} of [
+        {width: 768, height: 900},
+        {width: 1024, height: 900},
+        {width: 1280, height: 900},
+    ]) {
+        test(`${width}x${height}`, async ({page}) => {
+            await page.setViewportSize({width, height});
+            await tuiGoto(page, `/services/breakpoint-service`);
+            const example = page.locator(
+                `tui-doc-example[heading="Basic"] [automation-id=tui-doc-example]`,
+            );
+
+            await example.waitFor({state: `visible`});
+            await example.scrollIntoViewIfNeeded();
```

Comment:
Did it actually needed ?
I suppose that `toHaveScreenshot` command do it automatically 🤔

Reply:
Makes sense, will change this